canonical-webteam/example-js
# Add syntax highlighting

## Done
Bundled highlight.js into example.js. Parse the code block with highlight.js before placing on the page. Loop through the styles and apply them to the parsed markup.

## QA
- Pull this branch
- Run 
```bash
npm install
npm run build
npm start
```
- Go to http://127.0.0.1:8080/testing/
- Check there is syntax highlighting

## Details
Fixes https://github.com/canonical-webteam/example-js/issues/11
